commands/create-account.js

```diff
@@ -3,10 +3,18 @@ const exitOnError = require('../utils/exit-on-error');
 const connect = require('../utils/connect');
 const { KeyPair } = require('near-api-js');
 const eventtracking = require('../utils/eventtracking');
+const NEAR_ENV_SUFFIXES = {
```

Comment:
This for sure shouldn't be hardcoded in shell. IMO makes sense to pass this as `--masterAccount` from config/options, as they have to match basically.

Reply:
Bump on this. We could also get from config, then finally set a default since network is set there.

Reply:
Let's chat about this. I'm not sure what is being suggested here. Adding a key for `TLA` to the `config.js`?
As a reminder, since I've taken a while to get back to this, what we're aiming to accomplish is this:
**A user tries to create an account with a betanet account while connected to testnet.**
(This PR has a failure for when the TLA of the new account and master account are off, so that's not the issue at hand.)
If we're going to look at the network the user is connected to and say, "I see you're on testnet (aka `networkId` of **default**) but you're trying to create an account with a .betanet username, watch out"
also, "I see you're on production (aka `networkId` of **mainnet**) but are not using a .near username, watch out"
And since mainnet can have custom TLAs this is going to be just a warning, I believe.
Since the `config.js`'s `networkId` doesn't correspond to the TLA, what's the change here?
Do we change `config.js`'s `default` to be `testnet`?

Reply:
okay @vgrichina and @potatodepaulo I think I pushed up a change that will make sense here. Added `tla` key to the `config.js` file. Since we're in `strict` mode, I had to add another (hidden) flag into yargs to accept it. Then I check to see whether the network has a specified TLA, and if it does, check against the `masterAccount` when doing `create_account`

Reply:
> Since we're in strict mode, I had to add another (hidden) flag into yargs to accept it. 

I see no need to hide argument for this, if it's in config – it makes sense to document it as argument. It's pretty convenient to have everything set in config overridable from command line.

Also as commented here (https://github.com/near/near-shell/pull/332#discussion_r428976098) I think we can use the same `--masterAccount` key for this, as I don't see why you'd want to set these differently.

If we want to use it to make difference between master account used directly and the one used through helper – maybe it should be smth like `--helperAccount` to make clear it is related? As it for example can be e.g. `stardust.near` if Stardust is hosting their own helper to create accounts for users, but doesn't want to purchase TLD.

Reply:
I like the idea of removing the `hide` from `tla`
While I'm at it, I'm removing all the hidden flags, pushing up a commit soon.